bread/layout/components/datatable.py

```diff
@@ -89,6 +89,7 @@ def __init__(
         spacing="default",
         orderingurlparameter="ordering",
         zebra=False,
+        is_inlineformset=False,
     ):
         """columns: list of DataTableColumn(header, cell, sortingname, enable_row_click)
         row_iterator: python iterator of htmlgenerator.Lazy object which returns an iterator
```

Comment:
```suggestion
        row_iterator: python iterator or htmlgenerator.Lazy object which returns an iterator or a string which we need to forward to a `FormsetField` constructor in case `is_inlineformset` is true
```
I am not a fan of parameters with multiple meanings anyway. Can we not just have two static factory-methods (which both call the constructor, which contains the shared logic)? One with the FormsetField string as parameter and one with the Iterator/Lazy[Iterator]?